front/src/services/filterService.ts

```diff
@@ -0,0 +1,41 @@
+export enum FilterType {
+    CHECKBOX = "CHECKBOX",
+    RANGE = "RANGE"
+}
+
+export interface Filter {
+    id: number,
+    name: string
+    type: FilterType,
+}
+
+export interface CheckboxFilter extends Filter {
+    value: string;
+}
+
+export interface RangeFilter extends Filter {
+    value: {
+        start: number,
+        end: number,
+    }
+}
+
+export function getAllFilters(): Filter[] {
+    const filters = [{
```

Comment:
plz rename to mockUseGetAllFilters and add case for CheckboxFilter with few values

Reply:
done